common/hooks/pre-pkg/06-shlib-provides.sh

```diff
@@ -45,7 +45,7 @@ collect_sonames() {
 hook() {
 	local _destdir32=${XBPS_DESTDIR}/${pkgname}-32bit-${version}
 
-	if [ -z "$shlib_provides" -a "${archs// /}" = "noarch" -o -n "$noshlibprovides" ]; then
+	if [ -z "$shlib_provides" -o -n "$noshlibprovides" ]; then
```

Comment:
Equivalent to old condition would be `[ -n "$noshlibprovides" ]`, but I do not get intent of its left side of `-o`.

Reply:
So old left side is "Noarchs packges don't have shared libs, but if $shlib_provides is set, run the hook". Please change to `[ -n "$noshlibprovides" ]`.

Reply:
Oh, yeah, so it is, oops.